user/downloading-installing-upgrading/installation-guide.md

```diff
@@ -133,12 +133,30 @@ settings.
 ### Getting to the boot screen
 
 Just after you power on your machine, make the Qubes OS medium available to the
-computer by inserting your DVD or USB drive. Shortly after the Power-on
-self-test (POST) is completed, you should be greeted with the Qubes OS boot
-screen. 
+computer by inserting your USB drive.
+
+Then you'll need to power on the computer and immediately press a specific key to
+enter what's called the "BIOS" of you computer. The key to press varies wildly from
```

Comment:
What about users who are using (U)EFI instead of BIOS?

Reply:
In my UEFI BIOS it's pretty much the same. Do you have a different experience? Perhaps I should look at more distro's advice

Reply:
> In my UEFI BIOS it's pretty much the same. Do you have a different experience? Perhaps I should look at more distro's advice

Well, it's more about the language. Even if the experience is the same, the reader won't know that unless we tell them. If we just say "BIOS," then someone using UEFI may think, "Aw, crap! My computer uses UEFI, not BIOS, so this whole guide doesn't apply to me! :frowning_face:" So it's better to say "'BIOS' or 'UEFI'" or somehow indicate that we're talking about both.